client/verta/tests/test_utils/test_time_utils.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime, timedelta
+
+from hypothesis import given
+import hypothesis.strategies as st
+import pytest
+from verta._internal_utils import time_utils
+from ..time_strategies import millis_uint64_strategy, millis_timedelta_strategy
+
+
+def _check_positive_millis(millis):
+    assert isinstance(millis, int) and millis > 0
+
+
+class TestEpochMillis:
+    def test_none(self):
+        assert time_utils.epoch_millis(None) is None
+
+    def test_naive_dt(self):
+        naive_dt = datetime.now()
+        assert naive_dt.tzinfo is None
+        with pytest.warns(UserWarning):
+            millis = time_utils.epoch_millis(naive_dt)
+            _check_positive_millis(millis)
```

Comment:
Which of these two lines is expected to raise the `UserWarning`? Could we separate them?